File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ListenfreeSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await ListenfreeSDK.test()
    equal(null !== testsdk, true)
  })

})

File: rb/test/exists_test.rb
# Listenfree SDK exists test

require "minitest/autorun"
require_relative "../Listenfree_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ListenfreeSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
